mixin/dashboards/bucket_replicate.libsonnet

```diff
@@ -6,31 +6,32 @@ local g = import '../lib/thanos-grafana-builder/builder.libsonnet';
     jobPrefix: error 'must provide job prefix for Thanos Bucket Replicate dashboard',
     selector: error 'must provide selector for Thanos Bucket Replicate dashboard',
     title: error 'must provide title for Thanos Bucket Replicate dashboard',
+    namespaceLabel: error 'must provide namespace label', 
   },
   grafanaDashboards+:: {
-    'bucket_replicate.json':
+    'thanos-bucket_replicate.json':
```

Comment:
What is the motivation behind add `thanos` prefix? Did it break something?

Reply:
When using with other mixins like the kubernetes-mixin. It makes sense to have a prefix as the files are in the same directory (the generated json files). For `bucket_replicate.json` its fine but for some names like `overview.json` it names sense to have some prefix.

The kubernetes-mixin is also following the approach of adding `k8s-` as a prefix
https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/dashboards/resources/node.libsonnet#L32

What do you think?